flask_calendar/settings.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+from types import SimpleNamespace
+import inspect
+import importlib.util
+
+
+config = None
+
+
+def load_file(config_name: str = "config.py"):
```

Comment:
doesn't mypy complains about missing a return value `...) -> None:`?

Reply:
I like to keep things simple regarding configuration but as long as we don't build a custom config, no problem 👍

Reply:
> doesn't mypy complains about missing a return value `...) -> None:`?

Yep, they were passing, both in virtualenv and docker. I will fix it.